README.md

```diff
@@ -379,6 +379,7 @@ The default values for each rule are described below.
 | no_summary                   | warning |
 | no_array_responses           | error   |
 | parameter_order              | warning |
+| operation_id_naming_convention | warning |
```

Comment:
Sorry I missed this before - this isn't the only place in the README that the rule should be documented. See the other rules for examples